test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/Import/SqlServerFhirDataBulkOperationTests.cs

```diff
@@ -32,87 +22,42 @@ namespace Microsoft.Health.Fhir.Shared.Tests.Integration.Features.Operations.Imp
             operationsConfiguration.Value.Returns(new OperationsConfiguration());
-
-            _schemaInformation = new SchemaInformation(SchemaVersionConstants.Min, SchemaVersionConstants.Max);
-            _schemaInformation.Current = SchemaVersionConstants.Max;
-            _sqlServerFhirDataBulkOperation = new SqlImportOperation(_fixture.SqlConnectionWrapperFactory, operationsConfiguration, _fixture.SchemaInformation, NullLogger<SqlImportOperation>.Instance);
+            operationsConfiguration.Value.Import.DisableOptionalIndexesForImport = true;
+            _reindexer = new SqlImportReindexer(_fixture.SqlServerFhirDataStore, _fixture.SqlConnectionWrapperFactory, operationsConfiguration, NullLogger<SqlImportReindexer>.Instance);
         }
 
         [Fact]
-        public async Task GivenUnclusteredIndexes_WhenDisableIndexes_ThenIndexShouldBeChanged()
-        {
-            List<(Table table, Index index, bool pageCompression)> indexeRecords = new List<(Table table, Index index, bool pageCompression)>();
-            indexeRecords.AddRange(_sqlServerFhirDataBulkOperation.IndexesList());
-            indexeRecords.AddRange(_sqlServerFhirDataBulkOperation.UniqueIndexesList());
-            (string tableName, string indexName, bool pageCompression)[] indexes = indexeRecords.Select(indexRecord => (indexRecord.table.TableName, indexRecord.index.IndexName, indexRecord.pageCompression)).ToArray();
-            foreach (var index in indexes)
-            {
-                string compressionBefore = await GetIndexCompression(index.indexName, index.tableName);
-                bool isDisabled = await GetIndexDisableStatus(index.indexName);
-                Assert.False(isDisabled);
-                await DisableIndex(index.tableName, index.indexName);
-                isDisabled = await GetIndexDisableStatus(index.indexName);
-                Assert.True(isDisabled);
-
-                // Can disable twice
-                await DisableIndex(index.tableName, index.indexName);
-                isDisabled = await GetIndexDisableStatus(index.indexName);
-                Assert.True(isDisabled);
-            }
-        }
-
-        private async Task<bool> GetIndexDisableStatus(string indexName)
-        {
-            SqlConnectionWrapperFactory factory = _fixture.SqlConnectionWrapperFactory;
-            using (SqlConnectionWrapper connection = await factory.ObtainSqlConnectionWrapperAsync(CancellationToken.None))
-            using (SqlCommandWrapper command = connection.CreateRetrySqlCommand())
-            {
-                command.CommandText = $"select is_disabled from sys.indexes where name = '{indexName}'";
-
-                return (bool)(await command.ExecuteScalarAsync(CancellationToken.None));
-            }
-        }
-
-        private async Task<string> GetIndexCompression(string indexName, string tableName)
+        public async Task EnsureEnableDisableIndexesWorks()
```

Comment:
@LTA-Thinking a nit for now, but prefer tests to have consistent naming

Reply:
This name perfectly reflects what this test does. Wha name do you suggest?